kolibri_explore_plugin/apps/default/metadata.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
:+1: . In the future we may also return an empty object from the Django view if opening the metadata file fails.